arangod/Metrics/ClusterMetricsFeature.cpp

```diff
@@ -199,18 +213,17 @@ void ClusterMetricsFeature::update() {
         });
   }
   if (leader->empty()) {
-    return repeatUpdate(1);  // invalid leader, immediately retry
+    return repeatUpdate(100);  // invalid leader, immediately retry
   }
-  auto rebootId = isData ? oldData.get("RebootId").getNumber<uint64_t>() : 0;
-  // We use `"0"` instead of `""` because we cannot parse empty string parameter
-  auto serverId = isData ? oldData.get("ServerId").copyString() : "0";
+  auto rebootId = oldData.get("RebootId").getNumber<uint64_t>();
+  auto serverId = oldData.get("ServerId").copyString();
   data.reset();
   metricsFromLeader(nf, cf, *leader, std::move(serverId), rebootId, version)
       .thenFinal([this](futures::Try<LeaderResponse>&& raw) mutable noexcept {
         if (wasStop()) {
           return;
         }
-        uint32_t timeoutMs = 1;  // invalid leader, immediately retry
+        uint32_t timeoutMs = 100;  // invalid leader, immediately retry
```

Comment:
I set 100 instead of 1.
Because I think repeat request every 0.1 seconds is often enough.
For cases when something goes wrong in our setup, for an example leader reelection because network partion/leader crash.

Reply:
Suggest to use here the configured timeout for metrics